src/Core/Utility/FunEx.re

```diff
@@ -2,3 +2,11 @@ let tap = (f, x) => {
   f(x);
   x;
 };
+
+let rec repeat = (~count, f) =>
+  if (count <= 0) {
+    ();
+  } else {
+    f();
+    repeat(~count=count - 1, f);
+  };
```

Comment:
```suggestion
  if (count > 0) {
    f();
    repeat(~count=count - 1, f);
  };
```

The else branch automatically evaluates to `unit` if omitted.

Reply:
Nice!